### ISSUE TITLE
Export BrowserTracker interface from @snowplow/browser-tracker

### ISSUE THREAD

**Issue by talbct** [snowplow/snowplow-javascript-tracker#929]
**Describe the bug**
`newTracker` returns `BrowserTracker`, but `BrowserTracker` is not exported, so we cannot use it properly in our TypeScript code.

**To Reproduce**
```
import {BrowserTracker} from '@snowplow/browser-tracker';
let tracker: BrowserTracker = newTracker('...', '...');
```

**Expected behavior**
No unexported type error.

**Desktop (please complete the following information):**
 - OS: macOS 11.1
 - TypeScript 4.2.3, Node 12.18.0

**Comment by paulboocock** [comment]
Thank you for the report. I'll resolve this before the final release.

**Comment by talbct** [comment]
Workaround for TypeScript 2.8+ -
`const tracker: ReturnType<typeof newTracker> = newTracker(...);`

**Comment by paulboocock** [comment]
This should be resolved in 3.0.0-beta.4

I'll leave this issue open until the final v3 release.

**Comment by talbct** [comment]
Thank you, your other comment about it not being really needed alleviated the need, but I am sure someone would need it. :)
